profiles/views.py

```diff
@@ -138,6 +143,7 @@ def post(self, request):
         work_experience = WorkExperience(
             user=request.user,
             organization=request.POST['organization'],
+            location=request.POST['location'],
```

Comment:
Praise: Thanks for catching that! I added it in so the CV would have one, but I forgot to add to the form!